quipucords/api/apps.py

```diff
@@ -9,4 +11,8 @@ class ApiConfig(AppConfig):
     name = "api"
 
     def ready(self):
-        """Mark server ready."""
+        """Initialize the scan manager when the API server is ready."""
+        if "runserver" in sys.argv:
+            from scanner.manager import reinitialize  # pylint: disable=C0415
+
+            reinitialize()
```

Comment:
I think you're right about this being unnecessary. This was the _first_ place I started to reinitialize the variable when I started this branch, oh, about two weeks ago, but since then I made the _other_ changes that may have made this redundant. I'll remove this and test a few things locally to confirm.